libp2p/pubsub/pubsub.py

```diff
@@ -154,7 +157,13 @@ def get_hello_packet(self) -> rpc_pb2.RPC:
         peer_id = stream.mplex_conn.peer_id
 
         while True:
-            incoming: bytes = await read_varint_prefixed_bytes(stream)
+            try:
+                incoming: bytes = await read_varint_prefixed_bytes(stream)
+            except (ParseError, IncompleteReadError) as error:
```

Comment:
When encountering these errors, we should handle 1) the stream is only closed(eof), then close the stream. 2) peer sent the wrong format of message, then reset.
https://github.com/libp2p/go-libp2p-pubsub/blob/master/comm.go#L38-L46